lib/Server.js

```diff
@@ -474,6 +478,13 @@ class Server {
         };
       }
 
+      if (
+        typeof options.client.reconnect === "undefined" ||
+        options.client.reconnect === true
```

Comment:
`true` should not have count of re connection, i.e.
- true - unlimited reconnections
- false - no reconnections
- number - count of reconnections